Makefile

```diff
@@ -229,18 +248,21 @@ build-packages-python-standalone: ## Build Python packages (standalone)
 
 .PHONY: build-packages-python-standalone
 
-start-development-no-link: build-development-image ## Start development environment
+start-development-no-link: build-development-image-non-root ## Start development environment
 
 	@ echo "Starting development environment..."
+	@ mkdir -p $(CURDIR)/build
 
 	docker run \
+		--name=open-space-toolkit-$(project_name)-dev-non-root \
 		-it \
 		--rm \
-		--privileged \
 		--volume="$(CURDIR):/app:delegated" \
+		--volume="$(HOME)/.ssh:/home/$(dev_username)/.ssh:ro" \
+		--volume="$(HOME)/.gitconfig:/home/$(dev_username)/.gitconfig:ro" \
 		--workdir=/app/build \
-		$(docker_development_image_repository):$(docker_image_version) \
-		/bin/bash
+		$(docker_development_image_repository)-non-root:$(docker_image_version) \
+		/bin/zsh
```

Comment:
We should probably be consistent with the shell used across the dockerfile. This just opens an interactive shell, but it's possible to install something with `bash` and it not be visible to `zsh` because they have different instantiations of the `PATH` variable.

I'm fine with using zsh, but if we go that route, maybe can put a `SHELL_PATH` variable at the top of the file and reference that everywhere